docs/tutorial/quickstart.md

```diff
@@ -0,0 +1,253 @@
+# Tutorial: Get started with oras-go v2
+
+This tutorial introduces the basics of managing OCI artifacts with the [oras-go v2](https://pkg.go.dev/oras.land/oras-go/v2) package.
+
+You'll get the most out of this tutorial if you have a basic familiarity with Go and its tooling. If this is your first exposure to Go, please see [Tutorial: Get started with Go](https://golang.org/doc/tutorial/getting-started) for a quick introduction.
+
+The tutorial includes the following sections:
+
+1. Connect to a remote repository.
+2. Show tags in the repository.
```

Comment:
```suggestion
2. Show tags in a repository.
```

Reply:
I think "the" is more appropriate in the context? It's showing the tags in THE repository of the first section.